sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/client_base.py

```diff
@@ -26,6 +26,12 @@ class ApiVersion(str, Enum):
     V7_0 = "7.0"
     V2016_10_01 = "2016-10-01"
 
+def _get_policies(config, **kwargs):
```

Comment:
I don't see this being used anymore, do we have another way of configuring `HttpLoggingPolicy` and adding it to the pipeline?